src/helix/core/HelixSpriteFluentApi.hx

```diff
@@ -88,4 +88,13 @@ class HelixSpriteFluentApi
         HelixState.current.add(sprite.textField);
         return sprite;
     }
+
+    public static function addKeyBind(sprite:HelixSprite, keys:Array<String>, callback):HelixSprite
```

Comment:
This can can be renamed to `onKeyDown` and the arguments can change to `sprite:HelixSprite, callback:Array<FlxKey>->Void`